SpatialGDK/Extras/schema/authority_intent.schema

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
```

Comment:
FYI - when changing schema please be sure to increment the value in `RequireSetup` to ensure schema are correctly copied when others update.